python/tskit/formats.py

```diff
@@ -252,6 +260,30 @@ def load_legacy(filename, remove_duplicate_positions=False):
     return ts
 
 
+def fix_legacy_kastore(src, dest):
+    with kastore.load(src) as store:
```

Comment:
This should go into ``load_legacy``, so we can load a pre version 13. Should go something like
```
data = dict(store)
data["mutations/time"] = np.zeros(data["mutations/node"].shape,  dtype=np.float64)
tables = tskit.TableCollection.fromdict(**data) # or something
tables.compute_mutation_times()
return tables.tree_sequence()
```

Reply:
I thought so too but was dissuaded by the comment in load_legacy: "This method is only intended to be used to read old format HDF5 files. "

Reply:
That comment is a remnant from when all the files were HDF5 and we just referred to the files as "HDF5" as a matter of course. I think load_legacy should cope with any old file format and return a tree sequence, making any choices that need to be made to bring it up to date.

Reply:
Fixed in https://github.com/tskit-dev/tskit/pull/672/commits/f5cf3d2184e834f5e07b8777866b399d37d82ac3